android/assets/jsons/translations/Dutch.properties

```diff
@@ -2242,9 +2241,9 @@ Hail. = Hail
 What do you want? = Wat wilt u?
 The Glory of Rome = The Glorie van Rome
  # Requires translation!
-The blessings of the gods be upon you, Caesar Augustus, emperor of Rome and all her holdings. Your empire was the greatest and longest lived of all in Western civilization. And your people single-handedly shaped its culture, law, art, and warfare like none other, before or since. Through years of glorious conquest, Rome came to dominate all the lands of the Mediterranean from Spain in the west to Syria in the east. And her dominion would eventually expand to cover much of England and northern Germany. Roman art and architecture still awe and inspire the world. And she remains the envy of all lesser civilizations who have followed. = 
+The blessings of the gods be upon you, Caesar Augustus, emperor of Rome and all her holdings. Your empire was the greatest and longest lived of all in Western civilization. And your people single-handedly shaped its culture, law, art, and warfare like none other, before or since. Through years of glorious conquest, Rome came to dominate all the lands of the Mediterranean from Spain in the west to Syria in the east. And her dominion would eventually expand to cover much of England and northern Germany. Roman art and architecture still awe and inspire the world. And she remains the envy of all lesser civilizations who have followed. = De zegeningen van de goden zijn op jou, Caesar Augustus, keizer van Rome en al haar bezittingen. Je rijk was de grootste en langst geleefde van alles in de westerse beschaving. En je mensen vormgeven zijn cultuur, wet, kunst en oorlogsvoering als geen ander, vóór of sindsdien. Door jarenlang glorieuze verovering kwam Rome om alle landen van de Middellandse Zee uit Spanje in het westen naar Syrië in het oosten te domineren. En haar heerschappij zou uiteindelijk uitbreiden om veel van Engeland en Noord-Duitsland te bedekken. Romeinse kunst en architectuur staan nog steeds en inspireren de wereld. En ze blijft de afgunst van alle mindere beschavingen die hebben gevolgd.
```

Comment:
Mijn taalgevoel is het niet eens met hoe je hier `vormgeven` gebruikt, gedeeltelijk omdat het een splitsbaar werkwoord is wat hier niet wordt gesplitst, en gedeeltelijk omdat het opeens tegenwoordige tijd is ipv verleden tijd zoals de rest van het stukje. Misschien iets als 'hebben vormgegeven'?
Verder is 'kwam te domineren' best wel een anglicisme, ik zou het zelf eerder iets als 'heeft veroverd' doen ofzo.
'still awe' zou ik zelf iets als 'boezemen nog steeds ontzag in' hebben vertaald